Packs/CortexResponseAndRemediation/Playbooks/silent-Playbook-Azure_AD_Administrator_Role_Assignment.yml

```diff
@@ -0,0 +1,2813 @@
+                    simple: modules.brand
+                  iscontext: true
+                right:
+                  value:
+                    simple: Microsoft Graph User
+                ignorecase: true
+            - - operator: isEqualString
+                left:
+                  value:
+                    simple: modules.state
+                  iscontext: true
+                right:
+                  value:
+                    simple: active
+                ignorecase: true
+            transformers:
+            - operator: count
+            - operator: If-Elif
+              args:
+                conditions:
+                  value:
+                    simple: |-
+                      [{
+                        "condition": "#{.} > 0",
+                        "return": "Disable Assignor User"
+                      }, {
+                        "default": "Confirm"
+                      }]
+                flags: {}
+        - complex:
+            root: modules
+            filters:
+            - - operator: isEqualString
+                left:
+                  value:
+                    simple: modules.brand
+                  iscontext: true
+                right:
+                  value:
+                    simple: Microsoft Graph User
+                ignorecase: true
+            - - operator: isEqualString
+                left:
+                  value:
+                    simple: modules.state
+                  iscontext: true
+                right:
+                  value:
+                    simple: active
+                ignorecase: true
+            transformers:
+            - operator: count
+            - operator: If-Elif
+              args:
+                conditions:
+                  value:
+                    simple: |-
+                      [{
+                        "condition": "#{.} > 0",
+                        "return": "Disable Assignee User"
+                      }, {
+                        "default": "Confirm"
+                      }]
+                flags: {}
+        fieldassociated: ""
+        placeholder: ""
+        tooltip: ""
+        readonly: false
+      title: "🔍 Analyst Action Required 🔍"
+      description: |-
+        It is recommended to review the information displayed in this task.
+        **Review and take action if needed:**
+      sender: Your SOC team
+      expired: false
+      totalanswers: 0
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "67":
+    id: "67"
+    taskid: 80026f48-de41-4066-a2de-95c0182df4ad
+    type: condition
+    task:
+      id: 80026f48-de41-4066-a2de-95c0182df4ad
+      version: -1
+      name: Check Assignor Type
+      description: If the **assignor's identity** is a *USER*, we can revoke their session and disable the user if necessary. However, for an *APPLICATION* identity, we cannot currently perform dedicated remediation steps
```

Comment:
If the **assignor's identity** is a *USER*, you can revoke their session and disable the user if necessary. However, for an *APPLICATION* identity, you cannot perform dedicated remediation steps.